src/app/pages/observations/observation-list.component.html

```diff
@@ -1,14 +1,36 @@
-<otp-tabs [tabs]="tabs"></otp-tabs>
 <div class="content">
   <div>
-    <a [routerLink]="[private/observations/new]">New observation</a>
+    <a href="private/observations/new">New observation</a>
```

Comment:
You should use `routerLink`: https://angular.io/docs/ts/latest/guide/router.html#!#basics-router-links